rocAL/rocAL/source/augmentations/effects_augmentations/node_snp_noise.cpp

```diff
@@ -21,42 +21,56 @@ THE SOFTWARE.
 */
 
 #include <vx_ext_rpp.h>
-#include <VX/vx_compatibility.h>
 #include "node_snp_noise.h"
 #include "exception.h"
 
 
 SnPNoiseNode::SnPNoiseNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs),
-        _sdev(SDEV_RANGE[0], SDEV_RANGE[1])
+        _noise_prob(NOISE_PROB_RANGE[0], NOISE_PROB_RANGE[1]),
+        _salt_prob (SALT_PROB_RANGE[0], SALT_PROB_RANGE[1]),
+        _noise_value(NOISE_RANGE[0], NOISE_RANGE[1]),
+        _salt_value(SALT_RANGE[0], SALT_RANGE[1])
 {
```

Comment:
Move the brackets to prev line
_salt_value(SALT_RANGE[0], SALT_RANGE[1]) { }

Follow this for other files too